pkg/apis/policy/validation/validation_test.go

```diff
@@ -138,12 +138,10 @@ func TestValidatePodDisruptionBudgetUpdate(t *testing.T) {
 			generations: []int64{int64(2), int64(3)},
 			specs: []policy.PodDisruptionBudgetSpec{
 				{
-					MinAvailable:   &c1,
-					MaxUnavailable: &c2,
+					MinAvailable: &c1,
 				},
 				{
-					MinAvailable:   &c1,
-					MaxUnavailable: &c2,
+					MinAvailable: &c1,
```

Comment:
why is MaxUnavailable removed ?

Reply:
It's invalid to set both `MinAvailable` and `MaxUnavailable`, so this test case should not have been trying to do so (it's not intended to test an error condition). The flawed test case incidentally used to work because we did not validate `.spec` at all during updates, since validating spec on creation and enforcing immutability made that unnecessary.